### PR TITLE
[GST-2149] Ported Mosquitte role from ansible-roles

### PR DESCRIPTION
**Pull request by lobsterdore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,142 @@
...
+```YAML
+- name: Install Mosquitto with username/password file
+  hosts: sandbox
+
+  roles:
+    - name: sansible.mosquitto
+      mosquitto:
+        settings:
+          allow_anonymous: "false"
+          password_file: /etc/mosquitto/passwd
```
Comment (racer1680) [inline README.md]: passwd and path to the file...? Is it encrypted?

Comment (racer1680) [inline README.md:134]: Would it be possible not to mention path here?

Comment (lobsterdore) [reply README.md:134]: Why? It's just a readme file.

**Comment by racer1680** [comment]
LGTM

**Pull request merged**